### PR TITLE
Netcore: Hotfix for PublishedSnapshotService initialization

### PR DESCRIPTION
**Pull request by rustybox**
Resolves issue from https://github.com/umbraco/Umbraco-CMS/pull/9415#issuecomment-733586568

All tests green for me locally.

### CODE REVIEW

File: src/Umbraco.PublishedCache.NuCache/PublishedSnapshotService.cs [hunk 1]
```diff
@@ -84,13 +83,21 @@ internal class PublishedSnapshotService : PublishedSnapshotServiceBase
 
         //private static int _singletonCheck;
 
-        public PublishedSnapshotService(PublishedSnapshotServiceOptions options, IMainDom mainDom, IRuntimeState runtime,
-            ServiceContext serviceContext, IPublishedContentTypeFactory publishedContentTypeFactory,
-            IPublishedSnapshotAccessor publishedSnapshotAccessor, IVariationContextAccessor variationContextAccessor,
+        public PublishedSnapshotService(
+            PublishedSnapshotServiceOptions options,
```
Comment (JimBobSquarePants) [inline src/Umbraco.PublishedCache.NuCache/PublishedSnapshotService.cs:87]: If ever there was a sign that a class needs redesigned. This is it. 24 parameters!!!

Comment (bergmania) [reply src/Umbraco.PublishedCache.NuCache/PublishedSnapshotService.cs:87]: I think everyone agrees :P

**Comment by bergmania** [comment]
Thank you for the fast fix.. I will test it :)

File: src/Umbraco.PublishedCache.NuCache/PublishedSnapshotService.cs (hunk 1 above)
Comment (rustybox) [reply src/Umbraco.PublishedCache.NuCache/PublishedSnapshotService.cs:87]: c4jt321

**Comment by bergmania** [comment]
Seems to have fixed the issue! 💪

**Pull request merged**